[ADD] project: new top bar page (#11519)
odoo/documentation: content/applications/services/project/project_management/top_bar.rst

=======
Top Bar
=======

In project management, reviewing the various records and documents related to a project is often
necessary. Odoo Project’s **top bar** provides quick access to these essential resources. You can
customize each project's top bar to match its specific needs.

Configuration
=============

> To set up the top bar, go to the :guilabel:`Project` app, open an existing project, and click the
> top bar :icon:`fa-sliders` :guilabel:`(sliders)` button. This makes the top bar appear above the
> search bar. Click the sliders button at the top center of the screen to select which records you
> want to display, such as timesheets, sales orders, invoices, documents, dashboards, and more. You
> can add as many options as you need to your project’s top bar.

Reviewer: ```suggestion
To set up the top bar for a project, go to the :guilabel:`Project` app, click the project's card, then click the
top bar :icon:`fa-sliders` :guilabel:`(sliders)` button. In the bar that appears above the
search bar, click the icon:`fa-sliders` :guilabel:`(sliders)` button to select the records you
want to display, such as timesheets, sales orders, invoices, documents, dashboards, etc.
```
- When possible, we try to avoid sentences that just "state" what things look like and instead group them with actionable steps (cf "This makes the top bar appear").
- IMO the last sentence is not necessary

> .. image:: top_bar/top-bar.png
>    :align: center
>    :alt: Top bar selection menu

Reviewer: ```suggestion
.. image:: top_bar/top-bar.png
   :alt: Top bar selection menu
```

Creating custom views
=====================

You can also create your own buttons to access more specific views:

> #. Use the **search bar** with keywords, filters, and grouping options.

Reviewer: ```suggestion
#. Customize the view  with keywords, filters, and grouping options using the **search bar**.
```

> #. Click the top bar button and go to Save View.

Reviewer: ```suggestion
#. Click the :icon:`fa-sliders` :guilabel:`(sliders)` button in the top bar and select :guilabel:`Save View`.
```

> #. Name your view and check the box if you want to share your button with other users.

Reviewer: ```suggestion
#. Edit the default button name if necessary, then enable :guilabel:`Shared` if you want to share the button with other users.
```
- Even though there’s only one checkbox right now, I think naming it is still valuable (e.g., it helps users confirm they’re in the right place).
- It might also be helpful to clarify that the "view name" will actually be the button label, as "view" feels more technical and less practical for users.

> .. image:: top_bar/top-bar-custom.png
>    :align: center
>    :alt: Top bar customization

Reviewer: ```suggestion
.. image:: top_bar/top-bar-custom.png
   :alt: Top bar customization
```

Author: I removed the picture as you suggested, as the part about top bar is long enough on this page I think :)

> Using top bar
> =============
> 
> Clicking the buttons added to your top bar displays and allows you to edit related records without
> leaving the Project app. To return to your project’s tasks main view, click the :guilabel:`Tasks`
> button in the top bar.

Reviewer: I'm not sure this section is necessary as it's pretty straight-forward. Instead, I suggestrephrasing this a little bit and moving it after line 16. Something like this maybe:

```suggestion
You can then click the buttons to access the related records without leaving the Project app. To return to your project’s tasks' Kanban view, click the :guilabel:`Tasks` button in the top bar.
```
